README.rst

```diff
@@ -134,23 +134,23 @@ Pin Layout
 
 The following table shows the typical pin layout used:
 
-+-----------+----------+---------------------------------------------------------------+--------------------------+
-|           | PCD      | Arduino                                                       | Teensy                   |
-|           +----------+-------------+---------+---------+-----------------+-----------+--------+--------+--------+
-|           | MFRC522  | Uno / 101   | Mega    | Nano v3 |Leonardo / Micro | Pro Micro | 2.0    | ++ 2.0 | 3.1    |
-+-----------+----------+-------------+---------+---------+-----------------+-----------+--------+--------+--------+
-| Signal    | Pin      | Pin         | Pin     | Pin     | Pin             | Pin       | Pin    | Pin    | Pin    |
-+===========+==========+=============+=========+=========+=================+===========+========+========+========+
-| RST/Reset | RST      | 9 [1]_      | 5 [1]_  | D9      | RESET / ICSP-5  | RST       | 7      | 4      | 9      |
-+-----------+----------+-------------+---------+---------+-----------------+-----------+--------+--------+--------+
-| SPI SS    | SDA [3]_ | 10 [2]_     | 53 [2]_ | D10     | 10              | 10        | 0      | 20     | 10     |
-+-----------+----------+-------------+---------+---------+-----------------+-----------+--------+--------+--------+
-| SPI MOSI  | MOSI     | 11 / ICSP-4 | 51      | D11     | ICSP-4          | 16        | 2      | 22     | 11     |
-+-----------+----------+-------------+---------+---------+-----------------+-----------+--------+--------+--------+
-| SPI MISO  | MISO     | 12 / ICSP-1 | 50      | D12     | ICSP-1          | 14        | 3      | 23     | 12     |
-+-----------+----------+-------------+---------+---------+-----------------+-----------+--------+--------+--------+
-| SPI SCK   | SCK      | 13 / ICSP-3 | 52      | D13     | ICSP-3          | 15        | 1      | 21     | 13     |
-+-----------+----------+-------------+---------+---------+-----------------+-----------+--------+--------+--------+
++-----------+----------+-------------------------------------------------------------------------+--------------------------+
```

Comment:
there is a second table below, please add the new data there.
or rearrange the tables and move teensy to the second.
there are two tables to avoid scrolling.